src/test/acceptance/features/navigation.feature

```diff
@@ -17,6 +17,7 @@ Feature: Application process navigation is controlled
       | are you pregnant                      |
       | card address                          |
       | do you have children three or younger |
+      | add your childrens dates of birth     |
```

Comment:
Should be `enter your childrens dates of birth` to match other tests

Reply:
Yes, this will be done in another PR.